doc/02_Pattern sicurezza/05_accesso-del-fruitore/02_id_auth_soap_02.rst

```diff
@@ -1,5 +1,3 @@
-.. _id_auth_soap_02:
```

Comment:
@vintra73 questo riferimento dovrebbe permettere di linkare il paragrafo direttamente dall'url. Vedi controindicazioni?

Reply:
mi domando se non è superfluo considerato che i riferimenti hai paragrafi sono generati dalla build, vedi ad esempio https://docs.italia.it/italia/piano-triennale-ict/lg-modellointeroperabilita-docs/it/master/doc/01_Pattern%20interazione/05_pattern-bloccanti/02_block_soap.html#esempio

oppure ho visto male?